crypto/tbs/src/lib.rs

```diff
@@ -265,4 +268,24 @@ mod tests {
         let sig = unblind_signature(bkey, bsig);
         assert!(verify(msg, sig, pk));
     }
+
+    #[test]
+    #[should_panic]
```

Comment:
should specify the panic message
```suggestion
    #[should_panic(expected = "Not enough signature shares")]
```

Reply:
Done